packages/core/src/lib/lit-core.ts

```diff
@@ -116,6 +116,7 @@ const NETWORKS_REQUIRING_SEV: string[] = [
   LitNetwork.Habanero,
   LitNetwork.Manzano,
   LitNetwork.DatilTest,
+  LitNetwork.Datil,
```

Comment:
Is this reworking `datil-test`

Reply:
It's working on `datil-test` if that's what you meant?